totest-manager.py

```diff
@@ -268,13 +269,17 @@ def overall_result(self, snapshot):
 
                 if not ignored:
                     number_of_fails += 1
-                    if not labeled and len(refs) > 0 and not self.dryrun:
+                    if not labeled and len(refs) > 0:
                         data = {'text': 'label:unknown_failure'}
-                        self.openqa.openqa_request(
-                            'POST', 'jobs/%s/comments' % job['id'], data=data)
+                        if self.dryrun:
+                            logger.info("Would label {} as unknown".format(job['id']))
+                        else:
+                            self.openqa.openqa_request(
+                                'POST', 'jobs/%s/comments' % job['id'], data=data)
                 elif labeled:
+                    text = 'Ignored issue' if len(refs) > 0 else 'Ignored failure'
```

Comment:
How about putting a real openQA label in here as well so that we can distinguish these in openQA easier and potentially even visualize differently?

Reply:
label:ignored_failure